ground_truth_labeling_jobs/multi_modal_parallel_sagemaker_labeling_workflows_with_step_functions/sagemaker_ground_truth_workflows.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>I suggest we remove<code>AWS_ACCOUNT_ID</code>  - I don't think it's adding a lot of value here and forces users to look this up. Below when creating the input bucket, please remove this variable. I suggest you use a timestamp to create a unique name instead:</p><pre class="ql-syntax" spellcheck="false">str(int(time.time()))
</pre><p>If you use this, be sure you import time  above: <code>import time</code>  . </p><p> </p><p>I also suggest you use a boto3 session to look up the <code>AWS_REGION</code> rather than making the user add it manually:</p><pre class="ql-syntax" spellcheck="false">region = boto3.session.Session().region_name
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>The input bucket is controlled by cloudformation, so we need the account id to match unless we modify our cloudformation templates.</p><p> </p><p>Region update makes sense</p>